Makefile

```diff
@@ -51,7 +51,7 @@ update-submodules:
 	tools/update-submodules.sh
 
 clean:
-	rm -rf dist/build
+	rm -rf dist/build tmp
```

Comment:
We can't remove `tmp` though, I have been using it to put ubo-related stuff for years now -- maybe just remove `tmp/node_modules`?

Reply:
Done.